src/components/events-list/event-summary.js

```diff
@@ -1,44 +1,44 @@
+import { Table, Typography } from '@equinor/eds-core-react';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import RelativeToNow from '../time/relative-to-now';
-import WarningState from './warning-state';
 import EventType from './event-type';
-
+import WarningState from './warning-state';
+import RelativeToNow from '../time/relative-to-now';
 import eventModel from '../../models/event';
-import { isWarningEvent } from '../../utils/event-model';
-import { Table } from '@equinor/eds-core-react';
 
-const EventSummary = ({ event }) => {
-  return (
-    <>
-      <Table.Cell>
+const EventSummary = ({ event }) => (
+  <>
+    <Table.Cell>
+      <Typography
+        token={{
+          textTransform: 'capitalize',
+          fontWeight: 'unset',
+          fontSize: 'unset',
+        }}
+      >
         <RelativeToNow time={event.lastTimestamp} titlePrefix="Start" />
-      </Table.Cell>
-      <Table.Cell>
-        <EventType event={event}></EventType>
-      </Table.Cell>
-      <Table.Cell>
-        {event.involvedObjectKind}/{event.involvedObjectName}
-      </Table.Cell>
-      <Table.Cell className="wrap">
-        <span>
-          {event.reason} - {event.message}
-          {event.involvedObjectState &&
-            event.involvedObjectState.pod &&
-            event.involvedObjectState.pod.restartCount > 0 && (
-              <span>
-                . Restarted {event.involvedObjectState.pod.restartCount} times
-              </span>
-            )}
-        </span>
-      </Table.Cell>
-      <Table.Cell>
-        {isWarningEvent(event) && <WarningState event={event}></WarningState>}
-      </Table.Cell>
-    </>
-  );
-};
+      </Typography>
```

Comment:
instead of using `Typography` here, you can just change how `dateText` is formatted in `src/utils/datetime.js` (line 53 and 55).

Reply:
fixed